### PR TITLE
Sprint start Stepper component 

### PR DESCRIPTION
**Pull request by kejno**
sprint1

### CODE REVIEW

File: rslang/src/basicComponents/Notification/NotificationView.jsx [hunk 1]
```diff
@@ -30,12 +30,12 @@ MyApp.propTypes = {
...
 export default function IntegrationNotistack(props) {
   const classes = {
-    root: 'notification-root',
+    root: `notification-root notification_${props.variant}`,
   };
 
   const {
     afterClose = () => {},
-    duration = 5000,
+    duration = 50000,
```
Comment (DmitryLebetsky) [inline rslang/src/basicComponents/Notification/NotificationView.jsx]: You know that duration will be 50 seconds?

File: rslang/src/components/Savannah/Savannah.jsx [hunk 2]
```diff
@@ -85,7 +85,7 @@ class Savannah extends Component {
     error: text.ru.answersMistaken,
   };
 
-  state = { ...initialState };
+  state = JSON.parse(JSON.stringify(initialState))
```
Comment (DmitryLebetsky) [inline rslang/src/components/Savannah/Savannah.jsx]: What is the reason you did it?

File: rslang/src/components/Savannah/Savannah.jsx [hunk 3]
```diff
@@ -323,15 +321,15 @@ class Savannah extends Component {
   }
 
   newStartGameHandle = () => {
-    const state = { ...initialState };
+    const state = JSON.parse(JSON.stringify(initialState))
```
Comment (DmitryLebetsky) [inline rslang/src/components/Savannah/Savannah.jsx]: What is the reason you did it?

File: rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss [hunk 4]
```diff
@@ -1,32 +1,32 @@
...
-  
\ No newline at end of file
+}
+.mistakeTitle{
+  color: gray;
+}
+.Total{
+  padding: 0 10px;
+  margin-left: 10px;
+  color: #ffffff;
```
Comment (DmitryLebetsky) [inline rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss]: $color-light

File: rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss [hunk 5]
```diff
@@ -1,32 +1,32 @@
...
+  border-radius: 10px;
+}
+
+.error{
+  background: rgba(240, 87, 108, .7);
+}
+
+.success {
+  background: rgba(161, 240, 69, .7);
```
Comment (DmitryLebetsky) [inline rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss]: It's better to move this colors to constants, like $sprint-color-error, $sprint-color-success

File: rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss [hunk 6]
```diff
@@ -1,32 +1,32 @@
...
+
+.Title{
+  margin: 0 0 10px 43px;
+  color: rgb(82, 82, 82);
+}
+
+.Content{
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
```
Comment (DmitryLebetsky) [inline rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss]: Why not to use our mixins? @include flex(column, flex-start, flex-start)

File: rslang/src/components/Sprint/Components/StartGame/StartGame.jsx [hunk 7]
```diff
@@ -25,15 +29,18 @@ const StartGame = ({
...
 
   );
 };
 
 const StartGameUserForm = ({
-  handleChangeUserWords, handleCurrentGroup, onSubmitForm, token,
-}) => (
+  handleChangeUserWords, handleCurrentGroup, onSubmitForm, token, currentGroup,
+}) => {
+  console.log(currentGroup)
```
Comment (DmitryLebetsky) [inline rslang/src/components/Sprint/Components/StartGame/StartGame.jsx]: Don't forget to remove console.log later

File: rslang/src/components/Sprint/Components/StartGame/StartGame.jsx [hunk 8]
```diff
@@ -48,16 +55,29 @@ const StartGameUserForm = ({
...
-          handleCurrentGroup={handleCurrentGroup}
-          onSubmitForm={onSubmitForm}
+        <Stepper
+          defaultValue={currentGroup + 1}
+          onChangeCommitted={(event, value, ...args) => handleCurrentGroup(value - 1, ...args)}
+          step={null}
+          max={count.groups}
+          marks={generateStepperMarks(groups)}
+          className="sprint-levels-stepper"
+          label="Выберите уровень:"
```
Comment (DmitryLebetsky) [inline rslang/src/components/Sprint/Components/StartGame/StartGame.jsx:58]: You can take it from our constant text. (text.ru....)

File: rslang/src/components/Sprint/Sprint.jsx [hunk 9]
```diff
@@ -79,7 +79,7 @@ class Sprint extends Component {
 
   status = questionStatus;
 
-  state = initialState
+  state = JSON.parse(JSON.stringify(initialState))
```
Comment (DmitryLebetsky) [inline rslang/src/components/Sprint/Sprint.jsx]: Stringify and parse are mutually inverse

File: rslang/src/components/Sprint/Sprint.jsx [hunk 10]
```diff
@@ -193,12 +193,14 @@ class Sprint extends Component {
...
       clearTimeout(timerId);
-      this.setState({ isFinished: true });
+      this.setState({ isFinished: true, });
       saveSessionInfoToLocal(applicationThings.SPRINT);
+
     }
   }
 
   onClickHandler = (e) => {
+    console.log(initialState)
```
Comment (DmitryLebetsky) [inline rslang/src/components/Sprint/Sprint.jsx]: Don't forget to remove

File: rslang/src/components/Sprint/Sprint.jsx [hunk 11]
```diff
@@ -238,9 +240,11 @@ class Sprint extends Component {
   }
 
   onReloadGame = () => {
-    const state = { ...initialState };
-    this.setState({ ...state });
-    this.updateState();
+    const state = JSON.parse(JSON.stringify(initialState))
```
Comment (DmitryLebetsky) [inline rslang/src/components/Sprint/Sprint.jsx]: Stringify and parse are mutually inverse

File: rslang/src/scss/components/_notification.scss [hunk 12]
```diff
@@ -1,21 +1,27 @@
...
+    }
   }
 
-  .SnackbarItem-variantError-21 {
-    background-color: $color-error;
+  &.notification_error {
+    .MuiSnackbarContent-root,
+    .MuiPaper-root {
+      background-color: $color-error;
+    }
```
Comment (DmitryLebetsky) [inline rslang/src/scss/components/_notification.scss]: Please, don't change general styles. If you want, you can add your class to notification and then change styles by your own selector

**Review by DmitryLebetsky — APPROVED** [review]

**Review by hallovarvara — CHANGES_REQUESTED** [review]
Please change this element for one like in Prototype:
image

Dima made it in `unmess-repeating-feature`, you can get from there, I'll change styles.

**Pull request merged**